data/open-data-resources.yaml

```diff
@@ -0,0 +1,119 @@
+  contact: '<a href="http://alanhudson.info/"><span class="CharOverride-2">Alan Hudson</span></a>, Policy Director (Transparency &amp; Accountability) at ONE and <a href="https://okfn.org/about/team/"><span class="CharOverride-2">Jonathan Gray</span></a>, Director of Policy and Ideas at the Open Knowledge Foundation (<a href="https://twitter.com/jwyg"><span class="CharOverride-2">@jwyg</span></a>)</p>
+                  <p class="text">URL: <a href="http://followthemoney.net/"><span class="CharOverride-2">http://followthemoney.net/</span></a></p>'
+  url: '<a href="http://followthemoney.net/"><span class="CharOverride-2">http://followthemoney.net/</span></a>'                 
+  case_studies_url: 
+  database: 'https://docs.google.com/a/one.org/spreadsheets/d/1Rj47jJzF-lBoaF1_cvvY4rM9DIQ3Bw2b6dy-BEQuDno/edit#gid=1240396981'
+
+- title: '3. Open Data Partnership for Development (ODP4D) Case Studies: Open Data Institute (ODI), The World Bank and Open Knowledge Foundation'
+  image: "static/img/resources/Image10204.png"
+  who: 'Partnership between ODI, The World Bank and Open Knowledge Foundation'
+  what: '<p class="text">What: As noted in the project announcement, the overall objectives of the partnership are to: “support developing countries to plan, execute and run open data initiatives; to increase re-use of open data in developing countries; to develop an ecosystem of skills, support and peer-learning; and grow the base of evidence on the impact of open data for development.”<a href="https://theodi.org/odp4d-faqs"><span class="CharOverride-2">3</span></a></p>
+                <p class="text">Project activities include “scoping the state of open data; assessing the readiness of countries to open up – and use – their data; training government officials, other policy makers, and civil society; undertaking research and producing guidelines on the best use of open data; and producing case studies of impact.”<a href="http://www.worldbank.org/en/news/press-release/2013/09/18/new-partnership-seeks-bring-benefits-open-data-developing-countries"><span class="CharOverride-2">4</span></a></p>'
+  rationale: “There needs to be a credible evidence base focused on developing countries and socio-economic development in order to both win the support of officials and to better execute future open data initiatives.”<a href="https://theodi.org/odp4d-faqs"><span class="CharOverride-2">5</span>
+  number_of_case_studies: "13"
+  countries_covered: 'Burkina Faso, Mexico, Morocco, Kyrgyzstan, Moldova, Macedonia and South Africa'
+  contact: '<a href="https://theodi.org/team/tom-heath"><span class="CharOverride-2">Tom Heath</span></a>, Head of Research, ODI; (@tommyh)</p>
+                  <p class="text">Project URL: <a href="https://theodi.org/odp4d"><span class="CharOverride-2">https://theodi.org/odp4d</span></a'
+  case_studies_url: '<a href="https://theodi.org/case-studies"><span class="CharOverride-2">https://theodi.org/case-studies</span></a>'
```

Comment:
Ummm this does not appear to be a URL... ?

In general, for fields that can include HTML, I recommend adding `_html` to the end of the field name (e.g. `contact_html`) so that it's as clear as possible to anyone reading it, but it's up to you.